src/unix/fs.c

```diff
@@ -347,6 +348,9 @@ static int uv__fs_mkstemp(uv_fs_t* req) {
   if (req->cb != NULL)
     uv_rwlock_rdunlock(&req->loop->cloexec_lock);
 
+clobber:
+  if (r < 0)
+    *(path) = '\0';
```

Comment:
style nit: parens seem odd
```suggestion
    path[0] = '\0';
```